### PR TITLE
Functionality/dashboard

### PR DESCRIPTION
**Pull request by michaelputnam67**
# Description drawing

- initial dashboard functionality and structure
- dashboard testing
- deck routing

## Issues

Closes # 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [x] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?

- [x] console.log()
- [x] dev tools
- [x] functionality check through web browser

# Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have deleted all comments/console.log() after functionality and understanding is final
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers

### CODE REVIEW

File: cypress/e2e/dashboard.cy.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+describe("Dashboard", () => {
```
Comment (userigorgithub) [inline cypress/e2e/dashboard.cy.js:1]: Great testing. We can polish it up along with login/modal before turning it in too!

**Review by mrlandino — APPROVED** [review]
Nice work. We will polish up the code on the next refactor.

**Review by userigorgithub — APPROVED** [review]
Good work! I assume more merging coming up.

**Review by friotious — APPROVED** [review]
Testing looks good.

**Pull request merged**